runtime/development/src/lib.rs

```diff
@@ -1678,6 +1691,44 @@ impl pallet_liquidity_rewards::Config for Runtime {
 	type WeightInfo = ();
 }
 
+frame_support::parameter_types! {
+	pub const BlockRewardsDomain: RewardDomain = RewardDomain::Block;
+	pub const EpocDurationBlockRewards: BlockNumber = 2 * Period::get();
+
+	// TODO: Convert to common consts
+	#[derive(scale_info::TypeInfo, Debug, PartialEq, Clone)]
+	// TODO: Ensure uniqueness, currently abbreviates "blockrewards/collator"
+	pub const CollatorCurrencyId: u32 = u32::from_be_bytes(*b"br/c");
```

Comment:
Not sure if we want this, but there is a `CurrencyId` enum at `libs/types/src/tokens.rs` with all currencies used in `centrifuge-chain`. Maybe we can add this currency there in order to track all our currencies. 
Maybe @mustermeiszer can clarify this.

Reply:
Do we need to move this currency via some `fungibles::*` trait? If not, I am happy to not have this as part of the `enum CurrencyId` if yes lets add it like something like this

```rust
pub enum CurrencyId {
    ...
    RewardCurrency {
        id: [u8; 12] // Not sure about the format here
    }
}

```

Reply:
Thanks! Yes, we need to use fungibles for it

Reply:
> To clarify, it's not the currency given for reward. It's an emulated currency used to stake internally. I think a simple variant without content is enough.

So `RewardCurrency` should be entirely fine, right?

Reply:
My main Objektive is to but clutter our currency id enum with those staking/reward logic based currencies.

Reply:
Ok, gotcha, as a simple namespace 👍🏻. Although at least for now, we only have one currency of this kind.